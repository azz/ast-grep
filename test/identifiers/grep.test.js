import run from '..';

run(__dirname, 'foo');
run(__dirname, 'foo', { anonymous: true });
