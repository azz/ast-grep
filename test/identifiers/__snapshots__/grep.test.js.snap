// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`identifiers identifiers.js: frames 1`] = `
> 1 | foo();
    | ^^^
  2 | bar();
  3 | baz();
  4 | 

`;

exports[`identifiers identifiers.js: frames 2`] = `
> 1 | foo();
    | ^^^
  2 | bar();
  3 | baz();
  4 | 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  1 | foo();
> 2 | bar();
    | ^^^
  3 | baz();
  4 | 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  1 | foo();
  2 | bar();
> 3 | baz();
    | ^^^
  4 | 

`;

exports[`identifiers identifiers.js: raw 1`] = `

foo();

`;

exports[`identifiers identifiers.js: raw 2`] = `

foo();
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
bar();
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
baz();

`;
